app/routes/sign-up/success.tsx

```diff
@@ -0,0 +1,41 @@
+import { CheckCircleIcon } from '@heroicons/react/outline';
+import { Form } from '@remix-run/react';
+import { redirect } from '@remix-run/server-runtime';
+
+export async function action() {
+    return redirect('/');
+}
+
+export default function SuccessPage() {
+    return (
+        <div className="flex flex-col justify-center py-12 sm:px-6 lg:px-8">
+            <div className="mt-8 sm:mx-auto sm:w-full sm:max-w-md border-2 rounded-md border-lime-500">
```

Comment:
Small point, but can we use the "green" palette rather than "lime" for success? We're already using "green" for this and I find lime too bright.

Reply:
done.